content/en/docs/concepts/security/pod-security-policy.md

```diff
@@ -21,5 +21,5 @@ For a migration guide, see [Migrate from PodSecurityPolicy to the Built-In PodSe
 For more information on the removal of this API,
 see [PodSecurityPolicy Deprecation: Past, Present, and Future](/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/).
 
-If you are not running Kubernetes v{{< skew currentVersion >}}, check the documentation for
+If you are not running Kubernetes v{{< skew currentPatchVersion >}}, check the documentation for
```

Comment:
The original is fine; this statement applies to all patch versions in a minor release.